CONTRIBUTING.md

```diff
@@ -293,28 +288,22 @@ All wire protocol changes that may concern rolling upgrades should be documented
 
 ### Protobuf
 
-Akka includes a shaded version of protobuf `3` that is used for internal communication. To generate files,
+Pekko includes a shaded version of protobuf `3` that is used for internal communication. To generate files,
 run `protobufGenerate`. The generated files are put in each project's `src/main/java` and need to be committed.
 The generated files are automatically transformed to use the shaded version of protobuf.
 
 Generation depends on protoc `3.11.4` being on the path. See [protobuf project](https://github.com/protocolbuffers/protobuf#protocol-compiler-installation) for installation instructions, and
-[Protobuf.scala](https://github.com/akka/akka/blob/main/project/Protobuf.scala) for details of how to override
+[Protobuf.scala](https://github.com/apache/incubator-pekko/blob/main/project/Protobuf.scala) for details of how to override
 the settings for generation.
 
 ### Pull request requirements
 
 For a pull request to be considered at all, it has to meet these requirements:
 
 1. Regardless if the code introduces new features or fixes bugs or regressions, it must have comprehensive tests.
-1. The code must be well documented in the Lightbend's standard documentation format (see the 'Documentation' section below).
+1. The code must be well documented as per the existing documentation format (see the 'Documentation' section below).
 1. The commit messages must properly describe the changes. See further below.
 1. A pull request must be [linked to the issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue) it aims to resolve in the PR's description (or comments). This can be achieved by writing "Fixes #1234" or similar in PR description.
-1. All Lightbend projects must include Lightbend copyright notices.  Each project can choose between one of two approaches:
-
-    1. All source files in the project must have a Lightbend copyright notice in the file header.
-    1. The Notices file for the project includes the Lightbend copyright notice and no other files contain copyright notices.  See <https://www.apache.org/legal/src-headers.html> for instructions for managing this approach for copyrights.
```

Comment:
New rules for licenses:
* existing files with copyright statements must leave those copyright statements in tact
* new files should have an Apache license header instead

Reply:
Given that all source files will receive a significant change by the rename and will likely evolve away from the Lightbend original from there. Shouldn't we come up with a copy right header which merges the existing lightbend header with the ASF one.

To be approved by Apache legal of course.

Reply:
Note I had launched a thread on this exact subject on dev list
https://lists.apache.org/thread/xpc7yxdctkflntzvlxc9kw2yjbcth4n0

Reply:
my response is based on https://lists.apache.org/thread/z0r4sr4kdvmh52mlhc5porot2sh1wtok

there will be a need to have 2 copyrights in some files but it won't happen yet - the plan for next few months is minimal changes - we can agree what the 2 copyright header looks like - but it is not urgent - the 2 copyright header could be as simple as having the existing Lightbend license and then the Apache license after it

Reply:
If you open up an issue on the Legal JIRA, we can come up with a combined text there. Mixing the Akka header with the ASF one might mean some creativity. I also see that a lot of Akka headers seem to just be the Lightbend copyright statement; which again probably means some creativity.

Reply:
I will chase that.
I'm gathering some feedback here #38  on an initial proposal before I send it there.

Reply:
Will keep to the current text (as suggested by @pjfanning ) for now then. Can always update once we have more clarity.